### PR TITLE
Update/dbt Cloud quickstart

### PR DESCRIPTION
**Pull request by amychen1776**
In preparation of our rerecording of the Snowflake VHOL, I am updating the first half of the quickstart to use the new Snowflake UI to avoid any confusion.

### CODE REVIEW

File: site/package.json [hunk 1]
```diff
@@ -9,12 +9,12 @@
     "clean": "gulp clean"
   },
   "devDependencies": {
```
Comment (amychen1776) [inline site/package.json:11]: @jdanielmyers I updated my packages and this apparently changed this file. Should this be the case?

**Comment by amychen1776** [comment]
@jdanielmyers I wanted to check in and see if there was a possibility we could get this PR merged in by the end of the week? I was hoping to rerecord the VHOL next week 😁

**Comment by jdanielmyers** [comment]
thank you for the content!

**Comment by jdanielmyers** [comment]
Hey, there's actually a missing image that is causing the build to fail:

`assets/click_on_dbt_Partner_Connect.png`

File: site/sfguides/src/data_teams_with_dbt_cloud/data_teams_with_dbt_cloud.md [hunk 2]
```diff
@@ -136,13 +140,13 @@ Congratulations! You successfully tapped into a live data feed of Trade and FX r
...
+1. In the Snowflake UI, click on the home icon on the top left side. To access Partner Connect, click on your user and then Partner Connect. Make sure you're still set as the ACCOUNTADMIN role. 
+
+![Click on Partner Connect](assets/click_on_Partner_Connect_1.png)
 
-![Click on home](assets/click_on_home.png)
-![Click on Partner Connect](assets/click_on_Partner_Connect.png)
+If you're using the classic console, it will be on the top right. 
+![Click on Partner Connect](assets/click_on_Partner_Connect_2.png)
 
 2. Find the dbt Tile by typing into the "Search Partner Connect" search bar. Click on the dbt tile.
```
Comment (jdanielmyers) [inline site/sfguides/src/data_teams_with_dbt_cloud/data_teams_with_dbt_cloud.md:152]: assets/click_on_dbt_Partner_Connect.png is missing

**Comment by amychen1776** [comment]
@jdanielmyers Here is the new PR: https://github.com/Snowflake-Labs/sfquickstarts/pull/211

**Pull request merged**